builtInFunctions/esdtTransfer.go

```diff
@@ -121,7 +122,9 @@ func (e *esdtTransfer) ProcessBuiltinFunction(
 		return nil, err
 	}
 
-	if !check.IfNil(acntSnd) {
+	// or sender should be ESDTSCAddress in case of a sovereign scr
```

Comment:
// in cross shard ESDT transfer the sender account must be nil or sender should be ESDTSCAddress in case of a sovereign scr